docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,413 @@
+
+### Graphics
+
+We chose [`svg`][svg] over other image formats since clients can display it on
+any kind of screen resolution without requiring multi-layer files, or multiple
+files workarounds.
+
+Archive files
+-------------
+
+### `.meta/manifest.json`
+
+This is the only required file inside `.meta`, which declares information about
+the image, the target device type, and the publisher. This file may contain the
+following top level properties:
+
+- `publisher`: Information about the image publisher
+- `device`: Information about the image device type
+- `image`: Information about the image itself
+
+#### Publisher
+
+Here's an example of a real-world publisher manifest for RetroPie:
+
+```json
+{
+  "name": "RetroPie",
+  "url": "https://retropie.org.uk",
+  "supportUrl": "https://retropie.org.uk/forum/",
+  "colorScheme": {
+    "background": "#535760",
+    "text": "#FFFFFF",
+    "primary": "#5793db"
+  }
+}
+```
+
+Describe a publisher with any of the following properties:
+
+##### `name (String)`
+
+The display human-friendly name of the publisher.
+
+##### `url (String)`
+
+The main url of the publisher, usually the landing page.
+
+##### `supportUrl (String)`
+
+The publisher url where users can get general support.
+
+This could be a link to a forum, IRC room, troubleshooting page, support form,
+etc.
+
+##### `colorScheme (Object)`
+
+The publisher specific color scheme, if any.
+
+The client may allow a certain degree of branding based on this color scheme.
+
+You may declare the following colors, in hexadecimal format:
+
+- `background`: The background color
+- `text`: The text color
+- `primary`: The primary color
+
+#### Device
+
+Here's an example of a real-world device manifest for Raspberry Pi 3:
+
+```json
+{
+  "displayName": "Raspberry Pi 3",
+  "url": "https://www.raspberrypi.org/products/raspberry-pi-3-model-b/",
+  "arch": "armv7hf",
+  "slug": "raspberrypi3",
+  "slugAliases": [
+    "raspberry-pi3",
+    "rpi3"
+  ]
+}
+```
+
+Describe a device with any of the following properties:
+
+##### `displayName (String)`
+
+The human-friendly name of the device.
+
+##### `url (String)`
+
+The url to the landing page of the device.
+
+##### `arch (String[])`
+
+The device architecture.
+
+The current possible values are: `aarch64`, `amd64`, `armv5e`, `armv6`,
+`armv7hf`, and `i386`.
+
+##### `slug (String)`
+
+The device slug. Officially recognised slugs include:
+
+- `artik10`
+- `artik5`
+- `artik710`
+- `beaglebone-black`
+- `beaglebone-green-wifi`
+- `beaglebone-green`
+- `colibri-imx6dl`
+- `cybertan-ze250`
+- `edge`
+- `hummingboard`
+- `intel-edison`
+- `intel-nuc`
+- `nitrogen6x`
+- `odroid-c1`
+- `odroid-xu4`
+- `parallella`
+- `qemux86-64`
+- `qemux86`
+- `raspberry-pi`
+- `raspberry-pi2`
+- `raspberrypi3`
+- `ts4900`
+- `ts7700`
+- `via-vab820-quad`
+- `zynq-xz702`
+
+##### `slugAliases (String[])`
+
+The device slug aliases, if any.
+
+##### `compatibleWith (String[])`
```

Comment:
Wasn't there some discussion once about having "fat extended archives" that contained multiple images? Or maybe I'm just getting confused with something else.